src/inmanta/app.py

```diff
@@ -465,6 +465,19 @@ def export_parser_config(parser: argparse.ArgumentParser, parent_parsers: abc.Se
         action="store_true",
         default=False,
     )
+    parser.add_argument(
+        "--bypass-base-version-check",
+        dest="bypass_base_version_check",
+        help=(
+            "During partial exports, a check is performed to make sure the source code of Inmanta modules is identical to "
+            "the code used in the base version. If you don't have the option to perform a full export and if you are confident "
+            "that the code changed in such a way that it remains compatible, you can bypass this check with this option. "
```

Comment:
This is very verbose. I'd really try to keep it to the point, and be descriptive rather than speculative. e.g.

"Force handler code update during partial compile. This is otherwise only allowed for full compiles. Use with extreme caution, and only when confident that all code is compatible with previous versions."

And then the last part about the `--partial` flag.

It's 31 words versus 70, and I dare say it's just as clear.